pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "django-ses"
+version = "2.6.2"
+description = "A Django email backend for Amazon's Simple Email Service"
+authors = [
+    "Harry Marr <harry@hmarr.com>",
+    "Wes Winham <winhamwr@gmail.com>",
+    "Ross Lawley <ross.lawley@gmail.com>",
+    "Paul Craciunoiu",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    'Framework :: Django',
+    'Framework :: Django :: 2.2',
+    'Framework :: Django :: 3.2',
+    'Framework :: Django :: 4.0',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+license = "MIT"
+readme = "README.rst"
+repository = "https://github.com/django-ses/django-ses"
+include = ["example", "tests"]
```

Comment:
I think these are excluded from the build, in setup.py

Reply:
In fact they're included in the current builds. I think tests gets included automatically, but `example` is in the current builds because of this line:

https://github.com/django-ses/django-ses/pull/238/files#diff-41d5a52589e0480be9c099d2bba7a8135b8b0d71bcbb8df3582a8df1c2295003L4

I've never played with `MANIFEST.ini` files before though, so to be sure I checked the download from Pypi:

https://files.pythonhosted.org/packages/d9/78/331d2df82e0795722bdbb3918772bdd6c55955372fc0272f89817111944c/django-ses-2.5.0.tar.gz

If you open it up, it shows:

![Screenshot from 2022-03-21 14-05-43](https://user-images.githubusercontent.com/236970/159363710-1cf35d75-d845-4020-b876-6346263b561d.png)

Very happy to change this though. I don't really think it matters much either way, personally. :)

Reply:
Ah I saw this after, so it's just one folder that is newly included?

Reply:
No, the screenshot is the latest build on Pypi, so both tests and example were in there. I think the manifest file included the examples explicitly and I think the tests just get included by default. In any case, I think it's the same now as before.